console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/grid/EntityGrid.java

```diff
@@ -161,7 +162,11 @@ protected void configureEntityGrid() {
 
         //
         // Grid Data Load Listener
-        EntityGridLoadListener<M> entityGridLoadListener = new EntityGridLoadListener<M>(this, entityStore);
+        if (EntityFilterPanel.getSearchButton() != null && EntityFilterPanel.getResetButton() != null) {
```

Comment:
It looks like `EntityGridLoadListener` handles null values of `searchButton` and `resetButton` gracefully so, unless I'm missing something, I'm not sure that this if / else block is needed. You could just call the four parameters constructor and let the `EntityGridLoadListener` handle the cases where `EntityFilterPanel.getResetButton()` and `EntityFilterPanel.getResetButton()` are null. What do you think?